packages/material-ui/src/FormControl/FormControl.test.js

```diff
@@ -224,6 +224,16 @@ describe('<FormControl />', () => {
         setProps({ margin: 'dense' });
         expect(formControlRef.current).to.have.property('margin', 'dense');
       });
+
```

Comment:
This seems like it might be a good place for this additional test, but if there is a better way/location, let me know.